Sources/Data Model/DispatchEvents/BatchEvent.swift

```diff
@@ -104,7 +120,7 @@ struct DispatchEvent: Codable, Equatable {
     var tags: [String: AttributeValue]?
     var revenue: AttributeValue?
     var value: AttributeValue?
-
+    
```

Comment:
remove these extra spaces.

Reply:
this spacing is being added automatically by the new xcode, it follows the same spacing as the previous line.